cedar-policy-core/Cargo.toml

```diff
@@ -39,7 +40,7 @@ wasm-bindgen = { version = "0.2.82", optional = true }
 
 [features]
 # by default, enable all Cedar extensions
-default = ["ipaddr", "decimal"]
+default = ["ipaddr", "decimal", "protobufs"]
```

Comment:
Maybe better to not enable by default if we're calling it an experimental feature?